README.md

```diff
@@ -0,0 +1,92 @@
+# mptcp-check
+A website to check the validity of a mptcp connection
+
+The website is build as a Flask app behind a lighttpd server.
+Everything is package in a docker container.
+
+## installing
+you will need to create the too following scripts. I would recommend the following. Do not forget to replace the `email` and `url`
+
+`start.sh`:
+``` bash
+#!/bin/bash
+email= #the email used by certbot
+url= #your domain name
+path=/var/docker/mptcp-check
+
+docker run --rm --name certbot-init \
+            -v "$path/cert:/etc/letsencrypt:rw" \
```

Comment:
in bash, it is often recommended to use `{}` around variable names (`${path}`) to avoid typos (`$path-ok` is using the var `path` and add `-ok`, but if `_` is used, that's different), and clearly delimit the variables names.